programs/lib/linker/flexpret.ld

```diff
@@ -71,4 +77,16 @@ SECTIONS
     } > RAM
 
     __end = ABSOLUTE(.);
+
+    .heap : ALIGN(4) {
+        __sheap = ABSOLUTE(.);
+        __eheap = ABSOLUTE(DSPM_END - TOTAL_STACKSIZES);
+    } > RAM
+
+    .stack : ALIGN(4) {
+        __estack = ABSOLUTE(.);
+        __sstack =  ABSOLUTE(. + TOTAL_STACKSIZES);
+    } > STACK
+
+    PROVIDE(__stack_start$ = __sstack);
```

Comment:
These sections are added to be able to run more error handling/sanity checks in the `reset_handler` - the function that is run on boot.